### PR TITLE
Add Tweak: Retainer Notes

### PR DESCRIPTION
**Pull request by arcticnw**
This tweak adds a new "Notes" column to the Retainer window, allowing player to set short note for each retainer.

image

Individual notes and the width of the new column are configured through the configuration window.

image

Known issues: when the window is being opened or being closed, this tweak sometimes fights with Accurate Venture Times tweak (which uses the same hook). No clue how to fix it, but it works besides those half-a-second moments there.

If there are any questions, name's "Miri/Nesswen" in the goat discord

### CODE REVIEW

File: Tweaks/UiAdjustment/RetainerNotes.cs [hunk 1]
```diff
@@ -30,23 +33,21 @@ public class Configs : TweakConfig {
...
             }
         };
         #endregion
 
         public Configs Config { get; private set; }
 
-        public override string Name => "Retainer notes";
-        public override string Description => "Adds a note to the individual retainers in the retainer window.";
-        protected override string Author => "Nesswen";
+        private const int MaxRetainers = 12;
```
Comment (Caraxi) [inline Tweaks/UiAdjustment/RetainerNotes.cs:50]: Max retainers is 10

**Comment by Caraxi** [comment]
This is using the old way of extending the node list, I'd like to avoid adding any new tweaks directly changing the node list. Have a look at `PartyListStatusTimers` for a more updated example of adding a node

**Comment by arcticnw** [comment]
Yeah, I pretty much re-used code and style from AccurateVentureTimes tweak and ExtendedDesynthesisWindow tweak.

Anyways, review comments have been addressed + some additional minor fixes.

**Comment by arcticnw** [comment]
welp, seeing the 6.1 patch notes, this tweak is probably no longer necessary :D

**Pull request closed without merging**